src/generators.rs

```diff
@@ -644,66 +636,69 @@ pub fn binomial_tree_graph(
     })
 }
 
-/// Generate a directed binomial tree of order n recursively.
-/// The edges propagate towards right and bottom direction if ``bidirectional`` is ``false``
+/// Generate an directed heavy square graph. Fig. 6 of
+/// https://arxiv.org/abs/1907.09528.
+/// An ASCII diagram of the graph is given by:
 ///
-/// :param int order: Order of the binomial tree. The maximum allowed value
-///     for order on the platform your running on. If it's a 64bit platform
-///     the max value is 59 and on 32bit systems the max value is 29. Any order
-///     value above these will raise a ``OverflowError``.
-/// :param list weights: A list of node weights. If the number of weights is
-///     less than 2**order extra nodes with None will be appended.
+/// .. code-block:: console
+///
+///     ...       S   ...
+///        \     / \
+///        ... D   D   D ...
+///            |   |   |
+///        ... F-S-F-S-F-...
+///            |   |   |
+///        ... D   D   D ...
+///            |   |   |
+///        ... F-S-F-S-F-...
+///            |   |   |
+///            .........
+///            |   |   |
+///        ... D   D   D ...
+///             \ /     \
+///        ...   S       ...
+///
+/// NOTE: This function generates the four-frequency variant of the heavy square code.
```

Comment:
Same question about `.. note::` here.

Reply:
In https://github.com/Qiskit/rustworkx/pull/784/commits/5c3ea62972b3d9adcf62367ae55c14c030a8a270.